packages/contracts/contracts/THUSDOwner.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+
+import "./Interfaces/ITHUSDToken.sol";
+import "./Dependencies/CheckContract.sol";
+import "./Dependencies/Ownable.sol";
+
+contract THUSDOwner is Ownable, CheckContract {
+    ITHUSDToken private thusdToken;
+
+    address public governorBravoAddress;
+
+    constructor(
+        address _governorBravoAddress, 
+        address _thusdTokenAddress,
+        address _integrationsGuild
+    ) {
+        checkContract(_thusdTokenAddress);
+        thusdToken = ITHUSDToken(_thusdTokenAddress);
+
+        require(_integrationsGuild != address(0), "Integrations Guild address must be specified");
+        require(_governorBravoAddress != address(0), "Governor Bravo address must be specified");
+
+        governorBravoAddress = _governorBravoAddress;
+
+        _transferOwnership(_integrationsGuild);
+    }
+
+    function startRevokeMintList(address _account) external onlyOwner {
+        thusdToken.startRevokeMintList(_account);
+    }
+
+    function finalizeRevokeMintList() external onlyOwner {
+        thusdToken.finalizeRevokeMintList();
+    }
+
+    function transferThusdOwnershipToGovernorBravo(address _account) 
+        external 
+        onlyOwner 
+    {
+        require(_account == governorBravoAddress, "THUSDOwner: new owner must be Governor Bravo");
```

Comment:
Maybe make this function without parameter at all? `function transferThusdOwnershipToGovernorBravo()`?

Reply:
Good point, addressed in [refactor: remove param of transfer thusd ownership function](https://github.com/Threshold-USD/dev/pull/108/commits/3371b2b661ab1d729d168e59114e7555f7221421)